pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -1382,14 +1382,12 @@ func TestSelectNodesForPreemption(t *testing.T) {
 				cache.AddNode(&v1.Node{ObjectMeta: metav1.ObjectMeta{Name: name, Labels: map[string]string{"hostname": name}}})
 			}
 
-			predMetaProducer := algorithmpredicates.EmptyPredicateMetadataProducer
-
 			filterPlugin.failedNodeReturnCodeMap = filterFailedNodeReturnCodeMap
 			scheduler := NewGenericScheduler(
 				nil,
 				internalqueue.NewSchedulingQueue(nil, nil),
 				test.predicates,
-				predMetaProducer,
+				PredicateMetadata,
```

Comment:
nit: The `PredicateMetadata` is a one-liner. Seems that it can be removed. At least it doesn't need to be exported.

Reply:
done.